README.md

```diff
@@ -1,5 +1,35 @@
-# Firebase Functions Python SDK
+# Cloud Functions for Firebase Python SDK
 
-Welcome GitHub lurker! This repository is a work in progress. It will eventually let users create Cloud Functions for Firebase in Python. This feature is under very early development and not ready for testing. When it is ready, you can get first access by signing up for the [Firebase Alpha Program](https://services.google.cn/fb/forms/firebasealphaprogram/).
+The [`firebase-functions`](https://pypi.org/project/firebase-functions/) package provides an SDK for defining Cloud Functions for Firebase in Python.
```

Comment:
May be worth adding a "Status: Preview" note somewhere up here

Reply:
I don't see that in my diff, but I agree it is essential.  Might even put it in the title in parens.